src/Microsoft.ML.Transforms/Microsoft.ML.Transforms.csproj

```diff
@@ -56,6 +57,11 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
+    <Compile Update="TensorFlow\TensorGeneric.cs">
```

Comment:
Shall we move tensorflow to separate project + separate nuget package?

Reply:
Yeah,  I think everything that depends on TF (including the TensorFlowTransform) should be in a separate project + package.

Reply:
If you'd like to leave that part to me you can.  I can factor it out when I add the TF binaries.

Reply:
sounds good. lets address this as a separate follow up PR.

(marking as Pending for now)

---
In reply to: [212033224](https://github.com/dotnet/machinelearning/pull/704#discussion_r212033224) [](ancestors = 212033224)

Reply:
Eric will address this as a separate follow up PR

---
In reply to: [212001912](https://github.com/dotnet/machinelearning/pull/704#discussion_r212001912) [](ancestors = 212001912)

Reply:
This is now done.

---
In reply to: [212037611](https://github.com/dotnet/machinelearning/pull/704#discussion_r212037611) [](ancestors = 212037611,211834354)